integration_tests/src/main/python/spark_session.py

```diff
@@ -196,6 +196,12 @@ def is_databricks113_or_later():
 def is_databricks122_or_later():
     return is_databricks_version_or_later(12, 2)
 
+def is_supports_delta_lake_deletion_vectors():
```

Comment:
Name nit:
```suggestion
def supports_delta_lake_deletion_vectors():
```

Reply:
Thanks. I will rename this in the next PR.